### ISSUE TITLE
2.4.7 Focus Visible

### ISSUE THREAD

**Issue by svetlanaangelova** [samheather/iapt2group1#101]
- On homepage, Dashboard, View project and Browse/Search pages, when the image receives focus it is not visible.
